apps/opik-backend/src/main/java/com/comet/opik/api/filter/Operator.java

```diff
@@ -12,6 +12,7 @@ public enum Operator {
     STARTS_WITH("starts_with"),
     ENDS_WITH("ends_with"),
     EQUAL("="),
+    NOT_EQUAL("!="),
```

Comment:
Just make sure FrontEnd is happy with this syntax `!=` for the query param. From my side, it looks good.

Reply:
FE confirmed that they are good: https://comet-ml.slack.com/archives/C0768KH9MDJ/p1733391186958639